www/templates/html/MediaPlayer.tpl.php

```diff
@@ -127,13 +132,17 @@
                             });
                             var listItems = [];
                             for (var i = 0; i < track.entries.text.length; i++) {
-                                listItems.push($('<li>',  {"class": "highlight"})
+                                listItems.push($('<a>',  {
```

Comment:
This is going to make the created HTML invalid. You cannot put `<a>` directly into `<ul>`.